pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -383,10 +383,12 @@ func (az *Cloud) findServiceIPAddress(clusterName string, service *v1.Service, i
 		return "", err
 	}
 	if !existsLb {
-		return "", fmt.Errorf("Expected to find an IP address for service %s but did not", service.Name)
+		glog.V(4).Infof("Expected to find an IP address for service %s but did not. Assuming it has been removed", service.Name)
```

Comment:
set as glog.V(2) ? as default is 2

Reply:
Or Warningf ?

Reply:
This should be treated as normal message, changed to V(2)